### ISSUE TITLE
EnsureNuGetPackageBuildImports and <Import Project .targets> not removed. Bug or not?

### ISSUE THREAD

**Issue by MagicAndre1981** [hvanbakel/CsprojToVs2017#86]
With the new `PackageReference Include` entries, are the old NuGet entries (Import Project,  EnsureNuGetPackageBuildImports) obsolete or not?

I still see this in my converted csproj:

```
 <Import Project="..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" Condition="Exists('..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
    <PropertyGroup>
      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
    </PropertyGroup>
    <Error Condition="!Exists('..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets'))" />
  </Target>
  <PropertyGroup />
  <ItemGroup>
    <PackageReference Include="Microsoft.Bcl" Version="1.1.10" />
    <PackageReference Include="Microsoft.Bcl.Build" Version="1.0.21" />
    <PackageReference Include="Microsoft.Diagnostics.Tracing.EventSource.Redist" Version="1.1.28" />
  </ItemGroup>
```

**Comment by mungojam** [comment]
I think it is a duplicate of #66 which I may try and fix this evening

**Comment by MagicAndre1981** [comment]
ok, I overlooked this issue

**Comment by hvanbakel** [comment]
Fixed since #66 went in.
